balancer.go

```diff
@@ -83,11 +136,99 @@ func (info topicInfo) RoundRobin() map[string][]int32 {
 	return res
 }
 
+func rebalanceWithStrategy(topics map[string]TopicInfo,
+	rebalanceFunc func(TopicInfo) map[string][]int32,
+) map[string]map[string][]int32 {
+
+	res := make(map[string]map[string][]int32, 1)
+	for topic, info := range topics {
+		for memberID, partitions := range rebalanceFunc(info) {
+			if _, ok := res[memberID]; !ok {
+				res[memberID] = make(map[string][]int32, 1)
+			}
+			res[memberID][topic] = partitions
+		}
+	}
+	return res
+}
+
+// --------------------------------------------------------------------
+
+// StripedRebalancer implements StrategyStriped, balancing topics fairly
+// across consumers
+type StripedBalancer struct{}
+
+// AllMembersForGroup the unique set of group members for the given group info map.
+func AllMembersForGroup(group map[string]TopicInfo) []string {
```

Comment:
You could add a `type topicInfoGroup map[string]TopicInfo` helper and expose `MemberIDs() []string` and `Partitions() map[string][]int32` method on it